doc/man3/X509_VERIFY_PARAM_set_flags.pod

```diff
@@ -128,6 +133,9 @@ Security level 1 requires at least 80-bit-equivalent security and is broadly
 interoperable, though it will, for example, reject MD5 signatures or RSA keys
 shorter than 1024 bits.
 
+X509_VERIFY_PARAM_get0_host() returns the B<n>th expected DNS hostname that has
+been set using X509_VERIFY_PARAM_set1_host() or X509_VERIFY_PARAM_add1_host().
```

Comment:
Some discussion here about how you know the number of hosts would be good.

Reply:
I've just added the following hint:
To obtain all names start with n = 0 and increment n as long as no NULL pointer is returned.

Reply:
Update: 
I've simplified the calculation of `len1` and `len2` and adapted the commenting accordingly.
Anyway the semantics in case `ip->length` is not 2 * 4 or 2 * 16 is not really defined.